diff --git a/node_modules/@github/copilot-sdk/dist/session.js b/node_modules/@github/copilot-sdk/dist/session.js
index 1b92b45..ae64f5d 100644
--- a/node_modules/@github/copilot-sdk/dist/session.js
+++ b/node_modules/@github/copilot-sdk/dist/session.js
@@ -1,4 +1,4 @@
-import { ConnectionError, ResponseError } from "vscode-jsonrpc/node";
+import { ConnectionError, ResponseError } from "vscode-jsonrpc/node.js";
 import { createSessionRpc } from "./generated/rpc.js";
 class CopilotSession {
   /**
